seaborn/conftest.py

```diff
@@ -7,6 +7,29 @@
 import pytest
 
 
+def has_verdana():
```

Comment:
I hope this is an acceptable position for these lines. Perhaps we could even generalize the helper to `has_font(fontname)`.